ui_framework/doc_site/src/components/guide_page/guide_page_container.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+import { GuidePage } from './guide_page.jsx';
+
+function mapStateToProps(state, ownProps) {
+  return Object.assign({}, ownProps, {
```

Comment:
In general, never mutate (so never assign to `ownProps`). Also, what isn't immediately clear in react-redux is that `ownProps` is automatically injected, so no need to pass it in here.

In the Cloud UI code base we'd do this:

```
import { getSections } from '../../reducers';

const mapStateToProps = state => ({
  sections: getSections(state)
});
```

I've added a `getSections` helper. What we learned through experience was that grabbing into the `state` here makes it more difficult to move that state around in the tree later. So now we never grab into the state outside of the `reducers` folder, just rely on helpers like this. That has proven _very_ nice when "refactoring" the state layout, e.g. if you need an additional level or do some processing before returning state etc.